cmd/consensus/main.go

```diff
@@ -644,6 +592,100 @@ func main() {
 				return nil, fmt.Errorf("could not find latest finalized block and pending blocks: %w", err)
 			}
 
+			hotstuffModules = &consensus.HotstuffModules{
+				Forks:                nil,
+				Validator:            nil,
+				Notifier:             notifier,
+				Committee:            committee,
+				Signer:               signer,
+				Persist:              persist,
+				Aggregator:           nil,
+				QCCreatedDistributor: pubsub.NewQCCreatedDistributor(),
+			}
+
+			hotstuffModules, err = consensus.InitForks(
+				finalized,
+				node.Storage.Headers,
+				finalize,
+				hotstuffModules,
+				node.RootBlock.Header,
+				node.RootQC,
+			)
+			if err != nil {
+				return nil, err
+			}
+			hotstuffModules = consensus.InitValidator(mainMetrics, hotstuffModules)
+
+			voteProcessorFactory := votecollector.NewCombinedVoteProcessorFactory(committee, hotstuffModules.QCCreatedDistributor.OnQcConstructedFromVotes)
+
+			// TODO: add worker pool as separate component
+			workerPool := workerpool.New(4)
+
+			hotstuffModules.Aggregator, err = consensus.NewVoteAggregator(node.Logger, finalized, pending, hotstuffModules, workerPool, voteProcessorFactory)
+			if err != nil {
+				return nil, fmt.Errorf("could not initialize vote aggregator: %w", err)
+			}
```

Comment:
same as above comment
```suggestion
			aggregator, err := consensus.NewVoteAggregator(node.Logger, finalized, pending, hotstuffModules, workerPool, voteProcessorFactory)
			if err != nil {
				return nil, fmt.Errorf("could not initialize vote aggregator: %w", err)
			}
			hotstuffModules.Aggregator = aggregator
```